app/json/slate_configs.json

```diff
@@ -58,29 +58,49 @@
     ]
   },
   {
-    "id": "0737b00e-a21e-4875-a4c7-3e14926d4acf",
-    "displayName": "All Curated New Tab",
-    "description": "Curated items with syndicated that are at most a week old",
+    "id": "48e766be-5e96-46fb-acbf-55fee3ae8a28",
+    "displayName": "Editors' Picks",
+    "description": "Top 5 curated items without syndicated that are at most a week old",
     "experiments": [
       {
         "description": "default",
         "candidateSets": [
-          "35018233-48cd-4ec4-bcfd-7b1b1ccf30de"
+          "493a5556-9800-449f-8f8c-c27bb6c8c810"
```

Comment:
Do we need 2 experiments now?

Reply:
We didn't talk about changing this, and they're subtlety different when combined with Thompson sampling. I don't think it matters for our purposes, because we limit it to 5 in the end.